packages/matchbox/src/components/Text/tests/Text.test.js

```diff
@@ -20,20 +14,40 @@ describe('Text', () => {
     expect(wrapper).toMatchSnapshot();
```

Comment:
![Screen Shot 2020-05-20 at 11 10 12 AM](https://user-images.githubusercontent.com/3903325/82463248-9558a380-9a8a-11ea-97db-31544a041465.png)

Looks like the default isn't working correctly i think